packages/flutter/test/painting/paint_image_test.dart

```diff
@@ -52,7 +52,7 @@ void main() {
 
   test('debugInvertOversizedImages', () async {
     debugInvertOversizedImages = true;
-    expect(PaintingBinding.instance.window.devicePixelRatio != 1.0, true);
+    expect(PaintingBinding.instance.platformDispatcher.views.any((ui. FlutterView view) => view.devicePixelRatio > 1.0), isTrue);
```

Comment:
Instead of `views.any` here, would it make sense to just use `tester.view` here? Alternatively, if you're trying to keep the explicit reference to `PaintingBinding`, would `PaintingBinding.instance.platformDispatcher.views.single` work?

It seems a bit overkill to use `.any` when we're going to be assuming a single view during testing unless explicitly set up otherwise.

Reply:
> Instead of views.any here, would it make sense to just use tester.view here?

This is a regular `test`, not a `testWidgets` test, so we don't have access to the tester.

> PaintingBinding.instance.platformDispatcher.views.single work?

Yeah, this would work. However, I modeled the implementation here after the logic under test. For that logic to kick in we just need there to be at least one FlutterView with a greater dpr. It seemed odd to make the test more restrictive than the logic it is testing.

Reply:
Fair enough. I do feel like it'd be good to have a separate test case that validates this when there _are_ multiple views though. Granted we can't do that at the moment, but it'd be a good case to have since we are defaulting to a single view during tests.